src/utils/fetchClient.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const BASE_URL = 'https://mate.academy/students-api';
```

Comment:
move it to separate file - `utils/constants.ts` or use [.env](https://vite.dev/guide/env-and-mode) for it